Sources/Workspace/Workspace.swift

```diff
@@ -480,15 +490,12 @@ extension Workspace {
         try? fileSystem.removeFileTree(dataPath)
     }
 
-    /// Updates the current dependencies.
+    /// Resolves dependencies.
     ///
     /// - Parameters:
     ///     - diagnostics: The diagnostics engine that reports errors, warnings
     ///       and notes.
-    public func updateDependencies(
-        root: PackageGraphRootInput,
-        diagnostics: DiagnosticsEngine
-    ) {
+    fileprivate func resolveDependencies(root: PackageGraphRootInput, diagnostics: DiagnosticsEngine) throws -> (PackageGraphRoot, [(container: WorkspaceResolverDelegate.Identifier, binding: BoundVersion)], DiagnosticsEngine, PinsStore) {
```

Comment:
We should also provide labels to the return tuple.

Reply:
done